lib/fluent/plugin/buffer/chunk.rb

```diff
@@ -59,7 +60,7 @@ def initialize(metadata)
           @modified_at = Time.now
         end
 
-        attr_reader :unique_id, :metadata, :created_at, :modified_at, :state
+        attr_reader :unique_id, :chunk_id, :metadata, :created_at, :modified_at, :state
```

Comment:
Ah, i missed it... but it was used just once in newly added code.
So it seems not worth to add. If once it'll be added, it becomes a member of public API, and we must maintain it.
(In addition, adding stringified unique_id is bad idea because any internal code might change unique_id after instanciated. So it should be a `#chunk_id` method to return `dump_unique_id_hex(@unique_id)` at least.)